go/Makefile

```
@@ -181,7 +181,10 @@ integ-test: export TEST_FILTER = -skip "TestGlideTestSuite/TestModule" $(if $(te
 integ-test: __it
 
 # modules tests - run substask with default filter
+# Force modules-mode so the default client factories route through the TLS host slices,
+# which is required when the workflow only supplies tls-cluster-endpoints / tls-standalone-endpoints.
 modules-test: export TEST_FILTER = $(if $(test-filter), -run $(test-filter), -testify.m TestModule)
+modules-test: modules-mode = true
```

checkmake (0.3.2)

[warning] Target "modules-test" defined multiple times (lines 186 and 187).

(uniquetargets)